### PR TITLE
Image Processing Web Interface

### PR DESCRIPTION
**Pull request by ciorobeabianca**

### CODE REVIEW

File: src/main/java/com/bc/demoproj/DemoprojApplication.java [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-      Arrays.sort(beanNames);
-      for (String beanName : beanNames) {
-        System.out.println(beanName);
-      }
-
-    };
-  }
-
-}
+//package com.bc.demoproj;
```
Comment (dragos-petrascu) [inline src/main/java/com/bc/demoproj/DemoprojApplication.java]: We should avoid disabled source code by comments. Let's remove such a piece of code, instead.

File: src/main/java/com/bc/demoproj/HelloController.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 @RestController
 public class HelloController {
```
Comment (dragos-petrascu) [inline src/main/java/com/bc/demoproj/HelloController.java]: Let's keep image processing-related source code. Please remove "learning" source code.

File: src/main/java/com/bc/demoproj/uploadingfiles/FileUploadController.java [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    int i = 0;
+    int max = 400, rad = 10;
+    int a1 = 0, r1 = 0, g1 = 0, b1 = 0;
+    color = new Color[max];
+
+    int x = 1, y = 1, x1, y1, ex = 5, d = 0;
+
+    // Running nested for loops for each pixel
+    // and blurring it
+    for (x = rad; x < originalImage.getHeight() - rad; x++) {
```
Comment (dragos-petrascu) [inline src/main/java/com/bc/demoproj/uploadingfiles/FileUploadController.java]: Please check if duplicate code.

**Review by dragos-petrascu — APPROVED** [review]
Very good!

**Comment by ciorobeabianca** [comment]
my first pull request

**Pull request merged**